astropy/cosmology/parameters.py

```diff
@@ -50,9 +53,29 @@
 # in addition to the list above.  You also need to add them to the 'available'
 # list at the bottom of this file.
 
-# Planck 2018 paper VI v2
+# Planck 2018 paper VI
 # Unlike Planck 2015, the paper includes massive neutrinos in Om0, which here
 # are included in m_nu.  Hence, the Om0 value differs slightly from the paper.
+Planck18 = dict(
+    Oc0=0.2607,
+    Ob0=0.04897,
+    Om0=0.30966,
+    H0=67.66,
+    n=0.9665,
+    sigma8=0.8102,
+    tau=0.0561,
+    z_reion=7.82,
+    t0=13.787,
+    Tcmb0=2.7255,
+    Neff=3.046,
+    flat=True,
+    m_nu=[0., 0., 0.06],
+    reference=("Planck Collaboration 2018, 2020, A&A, 641, A6  (Paper VI),"
+               " Table 2 (TT, TE, EE + lowE + lensing + BAO)")
+)
+
+# Planck 2018 paper VI v2.  Identical to Planck18 above.
+# Warning: deprecated and will be removed in future versions.
```

Comment:
Do we have to actually emit a deprecation warning if this is used?

Reply:
I don't know.  It seems like it would be nice -- but how?  The existing deprecation annotations are for classes and attributes -- but this is neither.

Reply:
If it is passed into `default_cosmology`, then perhaps a warning could be emitted there, but I am not familiar with this sub-package.

Reply:
Me neither.

Who is the expert on astropy deprecation?

Reply:
Added DEPRECATED to the docstring.

There's no equals method for cosmology objects implemented.  Could check the docstring, but that feels a bit wierd.